src/service_name.json

```diff
@@ -36,203 +36,203 @@
   },
   {
     "Command": "az appservice",
-    "AzureServiceName": "Azure App Service",
```

Comment:
What's the purpose to remove "Azure" from the name? @dcaro, your thoughts?

Reply:
@jsntcy, In speaking with @dcaro, we felt it was redundant.  We are aligning with Azure PowerShell's product/service grouped reference TOC.

Reply:
Adding 'Azure' is an unnecessary and redundant information, since Azure CLI covers Azure services. Should it not be the case, we can use a prefix but this becomes an exception, not the norm.